### ISSUE TITLE
[iptorrents] (testing) Exception (iptorrents): Ssl error:1000007d:SSL routines:OPENSSL_internal:CERTIFICATE_VERIFY_FAILED  at /usr/ports/lang/mono/work/mono-6.8.0.105/external/boringssl/ssl/handshake_client.c:1132: The SSL connection could not be established, see inner exception.

### ISSUE THREAD

**Issue by KenwoodFox** [Jackett/Jackett#14488]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
2023-06-20 15:10:12 	Info 	Using HTTP Client: HttpWebClient
2023-06-20 15:10:12 	Info 	Mono version: 6.8.0.105 (6.8.0.105 Mon Mar  9 21:00:56 UTC 2020)
2023-06-20 15:10:12 	Info 	Using FlareSolverr: No
2023-06-20 15:10:12 	Info 	Using proxy: Disabled
2023-06-20 15:10:12 	Info 	App config/log directory: /usr/local/jackett
2023-06-20 15:10:12 	Info 	ThreadPool MaxThreads: 2400 workerThreads, 200 completionPortThreads
2023-06-20 15:10:12 	Info 	Running in Docker: No
2023-06-20 15:10:12 	Info 	Jackett variant: Mono
2023-06-20 15:10:12 	Info 	OS version: Unix 13.1.7.0 (64bit OS) (64bit process)
2023-06-20 15:10:12 	Info 	Environment version: 4.0.30319.42000 (/usr/local/lib/mono/4.5)
2023-06-20 15:10:11 	Info 	Jackett Data will be stored in: /usr/local/jackett
2023-06-20 15:10:11 	Info 	Starting Jackett v0.21.207
```

### Description

Sorry for posting this, it looks like other similar issues have been filed in relation to this problem, mainly looking at https://github.com/Jackett/Jackett/issues/12341 but it seems pretty stale/old, IIRC im using the latest versions i possibly can and im still having issues connecting with SSL.

I even tried spinning up a totally fresh install of jacket and no go, im not sure what changed on my end, infact my original jacket was very out of date! i just hadn't had to touch it in so long, but updating the base container, SSL, or jacket doesn't allow me to connect.

### Logged Error Messages

```text
Exception (iptorrents): Ssl error:1000007d:SSL routines:OPENSSL_internal:CERTIFICATE_VERIFY_FAILED at /usr/ports/lang/mono/work/mono-6.8.0.105/external/boringssl/ssl/handshake_client.c:1132: The SSL connection could not be established, see inner exception.
```

### Screenshots

![image](https://github.com/Jackett/Jackett/assets/31870999/5ff7e936-d66b-4ee6-ae9e-9affa420a539)

**Comment by ilike2burnthing** [comment]
What OS are you on?

Follow the troubleshooting steps here (ideally switching away from mono if possible) - https://github.com/Jackett/Jackett/wiki/Troubleshooting#your-cookie-did-not-work-the-ssl-connection-could-not-be-established

If using TrueNAS/FreeBSD - https://github.com/Jackett/Jackett/issues/12341#issuecomment-944936575

**Comment by KenwoodFox** [comment]
> What OS are you on?
> 
> Follow the troubleshooting steps here (ideally switching away from mono if possible) - [Jackett/Jackett/wiki/Troubleshooting#your-cookie-did-not-work-the-ssl-connection-could-not-be-established](https://github.com/Jackett/Jackett/wiki/Troubleshooting#your-cookie-did-not-work-the-ssl-connection-could-not-be-established)

Sorry about that! Im using FreeBSD 13, ill read over that wiki page thank you

**Comment by KenwoodFox** [comment]
I went ahead and tried using the fix here: https://github.com/Jackett/Jackett/issues/12341#issuecomment-938258497 but, no luck it seems, 
![image](https://github.com/Jackett/Jackett/assets/31870999/8a7216a6-bd78-46f0-b4c3-30de22575c04)

I tried restarting jackett too just to see if that's what was needed,

**Comment by KenwoodFox** [comment]
> 

I'm.. not sure I understand haha.. I keep re-reading your message but it doesn't make any sense, can you rephrase this?

**Comment by ilike2burnthing** [comment]
https://github.com/Jackett/Jackett/issues/12341#issuecomment-944936575
https://github.com/Jackett/Jackett/issues/12341#issuecomment-957585781
https://github.com/Jackett/Jackett/issues/12341#issuecomment-971888686

**Comment by KenwoodFox** [comment]
I'm sorry, I'm like, totally lost now. I don't know what you're trying to suggest to me, all of those are the same link and they point to a comment about blacklisting the ca serts?

I gave that a try and didn't have any luck, I don't understand the connection/bug at the moment. I don't want you to walk me through but I could use a little more direction, could you try rephrasing your earlier comment?

**Comment by ilike2burnthing** [comment]
The 3 links are to 3 different comments in on the same issue, none of which are the one you linked to earlier. No idea why they're not working for you, maybe try a private/incognito window. Otherwise, just read the comments under it and follow the instructions in them.
